home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/repository/NewsRepository.java

```diff
@@ -6,8 +6,8 @@
 import org.springframework.stereotype.Repository;
 
 /**
- * UserRepository is the interface that is needed
- * for interaction with Users in the database.
+ * NewsRepository is the interface that is needed
+ * for interaction with News in the database.
  *
  * @author Ihor Svyrydenko
  */
```

Comment:
This fix is ​​not relevant to my task. I simply corrected what the other guy has copied and not corrected before